From 7a8c7da972de7b5a546d5cc0743b7ecd559823ff Mon Sep 17 00:00:00 2001
From: B38613 <B38613@freescale.com>
Date: Tue, 12 Jun 2012 17:35:46 +0800
Subject: [PATCH 008/602] =?UTF-8?q?ENGR00180234-2=EF=BC=9Afix=20connectivity?=
 =?UTF-8?q?=20priority=20bug?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

MMS>ethernet>wifi>3G mobile
add ethernet priority between MMS and wifi.
value larger priority higher.

Signed-off-by: Jianzheng Zhou <B38613@freescale.com>
---
 .../overlay/frameworks/base/core/res/res/values/config.xml | 14 +++++++-------
 .../overlay/frameworks/base/core/res/res/values/config.xml | 14 +++++++-------
 2 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
index e20f9af..f531577 100755
--- a/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx53_smd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -84,14 +84,14 @@
     <string-array translatable="false" name="networkAttributes">
         <item>"wifi,1,1,1,-1,true"</item>
         <item>"mobile,0,0,0,-1,true"</item>
-        <item>"mobile_mms,2,0,2,60000,true"</item>
-        <item>"mobile_supl,3,0,2,60000,true"</item>
-        <item>"mobile_hipri,5,0,3,60000,true"</item>
-        <item>"mobile_fota,10,0,2,60000,true"</item>
-        <item>"mobile_ims,11,0,2,60000,true"</item>
-        <item>"mobile_cbs,12,0,2,60000,true"</item>
+        <item>"mobile_mms,2,0,3,60000,true"</item>
+        <item>"mobile_supl,3,0,3,60000,true"</item>
+        <item>"mobile_hipri,5,0,4,60000,true"</item>
+        <item>"mobile_fota,10,0,3,60000,true"</item>
+        <item>"mobile_ims,11,0,3,60000,true"</item>
+        <item>"mobile_cbs,12,0,3,60000,true"</item>
         <item>"wifi_p2p,13,1,0,-1,true"</item>
-        <item>"ethernet,9,9,0,-1,true"</item>
+        <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
     <!-- Component name of the service providing network location support. -->
diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
index 0cc7918..b263b04 100755
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -73,14 +73,14 @@
     <string-array translatable="false" name="networkAttributes">
         <item>"wifi,1,1,1,-1,true"</item>
         <item>"mobile,0,0,0,-1,true"</item>
-        <item>"mobile_mms,2,0,2,60000,true"</item>
-        <item>"mobile_supl,3,0,2,60000,true"</item>
-        <item>"mobile_hipri,5,0,3,60000,true"</item>
-        <item>"mobile_fota,10,0,2,60000,true"</item>
-        <item>"mobile_ims,11,0,2,60000,true"</item>
-        <item>"mobile_cbs,12,0,2,60000,true"</item>
+        <item>"mobile_mms,2,0,3,60000,true"</item>
+        <item>"mobile_supl,3,0,3,60000,true"</item>
+        <item>"mobile_hipri,5,0,4,60000,true"</item>
+        <item>"mobile_fota,10,0,3,60000,true"</item>
+        <item>"mobile_ims,11,0,3,60000,true"</item>
+        <item>"mobile_cbs,12,0,3,60000,true"</item>
         <item>"wifi_p2p,13,1,0,-1,true"</item>
-        <item>"ethernet,9,9,0,-1,true"</item>
+        <item>"ethernet,9,9,2,-1,true"</item>
     </string-array>
 
     <bool name="config_automatic_brightness_available">true</bool>
-- 
1.8.0

